etc/notebooks/examples/urth-viz-chart.ipynb

```diff
@@ -343,6 +395,15 @@
     "document.getElementById('c9').type='bar'\n",
```

Comment:
Since you changed the element's id above to `c10`... you need to do the same in this javascript cell. Right now it yields an error.